src/util/mongo_functions.js

```diff
@@ -82,8 +82,9 @@ function map_aggregate_objects() {
  * @this mongodb doc being mapped
  */
 function map_aggregate_chunks() {
+    const compress_size = this.compress_size || this.size;
     emit(['', 'compress_size'], this.compress_size);
```

Comment:
@romayalon use `compress_size` here too

Reply:
and not `this.compress_size`...
FYI @nimrod-becker

Reply:
ah!